### PR TITLE
Add tslint to Makefile

### PR DESCRIPTION
**Pull request by geremih**
So that `make lint` can be run before pushing, to ensure good style.
I have picked some rules for now which are specified in `tslint.json`.
Checkout https://github.com/palantir/tslint#supported-rules for more rules.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
     "typescript": "1.4",
     "browserify": "*",
     "decimal.js": "*"
```
Comment (perimosocordiae) [inline package.json]: need a comma at the end of this line

File: Makefile [hunk 2]
```diff
@@ -41,6 +43,9 @@ $(TSC):
 bower_components:
 	bower install
 
+lint:
+	$(TSLINT) $(foreach source,$(TSSOURCES), -f $(source))
```
Comment (perimosocordiae) [inline Makefile]: tslint accepts multiple files at a time, so we can simplify this to `$(TSLINT) -f $(TSSOURCES)`.

Comment (geremih) [inline Makefile]: `./node_modules/tslint/bin/lslint -f src/integer.ts src/interpreter.ts`
gives output only for `src/integer.ts` for me.

File: package.json (hunk 1 above)
Comment (geremih) [inline package.json]: Fixed!

File: Makefile (hunk 2 above)
Comment (perimosocordiae) [inline Makefile]: Ah, looks like a bug in tslint, then. Your workaround is fine.

**Comment by perimosocordiae** [comment]
Thanks!

**Comment by jvilk** [comment]
Excellent. I've been meaning to get tslint integrated on my other projects, but never got around to it. Thanks for the change!

**Pull request merged**